exts/loupe.simulation.beckhoff_bridge/loupe/simulation/beckhoff_bridge/BeckhoffBridge.py

```diff
@@ -1,41 +1,63 @@
-'''
+"""
   File: **BeckhoffBridge.py**
   Copyright (c) 2024 Loupe
   https://loupe.team
   
   This file is part of Omniverse_Beckhoff_Bridge_Extension, licensed under the MIT License.
   
-'''
+"""
 
 from typing import Callable
 import carb.events
 import omni.kit.app
+from ..common.RuntimeBase import get_stream_name
+from ..common.BridgeManager import BridgeManager, Manager_Events as ManEvents
```

Comment:
I think we decided common, needs to be renamed/moved here. Did we want to do that before merging?

Reply:
Ya, this should be fixed before we merge